File: clients/typescript-angular/API/Client/Order.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface Order {

        orderID: string;

        clOrdID?: string;

        clOrdLinkID?: string;

        account?: number;

        symbol?: string;

        side?: string;

        simpleOrderQty?: number;

        orderQty?: number;

        price?: number;

        displayQty?: number;

        stopPx?: number;

        pegOffsetValue?: number;

        pegPriceType?: string;

        currency?: string;

        settlCurrency?: string;

        ordType?: string;

        timeInForce?: string;

        execInst?: string;

        contingencyType?: string;

        exDestination?: string;

        ordStatus?: string;

        triggered?: string;

        workingIndicator?: boolean;

        ordRejReason?: string;

        simpleLeavesQty?: number;

        leavesQty?: number;

        simpleCumQty?: number;

        cumQty?: number;

        avgPx?: number;

        multiLegReportingType?: string;

        text?: string;

        transactTime?: date;

        timestamp?: date;
    }

}

File: clients/scala/src/main/scala/io/swagger/client/model/Order.scala
package io.swagger.client.model

import java.util.Date



case class Order (
  orderID: String,
  clOrdID: String,
  clOrdLinkID: String,
  account: Number,
  symbol: String,
  side: String,
  simpleOrderQty: Double,
  orderQty: Number,
  price: Double,
  displayQty: Number,
  stopPx: Double,
  pegOffsetValue: Double,
  pegPriceType: String,
  currency: String,
  settlCurrency: String,
  ordType: String,
  timeInForce: String,
  execInst: String,
  contingencyType: String,
  exDestination: String,
  ordStatus: String,
  triggered: String,
  workingIndicator: Boolean,
  ordRejReason: String,
  simpleLeavesQty: Double,
  leavesQty: Number,
  simpleCumQty: Double,
  cumQty: Number,
  avgPx: Double,
  multiLegReportingType: String,
  text: String,
  transactTime: Date,
  timestamp: Date)
  
